frontend/src/components/widgets/inputs/FontInput.vue

```diff
@@ -69,23 +58,27 @@
 			span {
 				color: var(--color-8-uppergray);
 			}
-
-			svg {
-				fill: var(--color-8-uppergray);
-			}
 		}
 	}
 
 	.menu-list .floating-menu-container .floating-menu-content {
 		max-height: 400px;
+		padding: 4px 0;
+
+		.spacer {
+			flex: 0 0 auto;
+		}
```

Comment:
This too. (Not sure about the `padding: 4px 0;`, but the `.spacer` isn't used here but you may need to move it to the other file.)